src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -880,6 +880,27 @@ interface i1
                 Diagnostic(ErrorCode.ERR_EventPropertyInInterface, "add"));
         }
 
+        [Fact]
+        public void CS0065ERR_EventNeedsBothAccessors_Interface05()
+        {
+            var text = @"
+public interface I2 { }
+
+public interface I1
+{
+    abstract event System.Action I2.P10;
```

Comment:
>abstract [](start = 4, length = 8)

Consider removing this modifier unless it is required to reproduce specific behavior.

Reply:
Fixed